data_structures/linked-list/ll_merge.py

```diff
@@ -15,25 +15,3 @@ def merge_lists(list_one, list_two):
             list_two.insert_before(current_two.val, current.val)
             current, current_two = current._next, current_two._next
         return list_two.head.val
```

Comment:
You should be returning a reference to `head` not `head.val` according to the spec.